ui/src/app/components/gke-app-configuration-panels/create-sas.spec.tsx

```diff
@@ -65,10 +65,10 @@ describe(CreateSAS.name, () => {
     registerApiClient(AppsApi, new AppsApiStub());
   });
 
-  it('should display a cost of $0.41 per hour when running', async () => {
```

Comment:
The default changed from 250 -> 150 GB for SAS hence the cost updated from 0.41 to 0.40